triggers/service/auth/src/test/scala/com/daml/auth/middleware/oauth2/TestFixture.scala

```diff
@@ -36,6 +40,17 @@ trait TestFixture
   lazy protected val server: OAuthServer = suiteResource.value._2
   lazy protected val serverBinding: ServerBinding = suiteResource.value._3
   lazy protected val middlewareBinding: ServerBinding = suiteResource.value._4
+  lazy protected val middlewareClient: Client = Client(
+    Client.Config(
+      authMiddlewareUri = Uri()
+        .withScheme("http")
+        .withAuthority(
+          middlewareBinding.localAddress.getHostName,
+          middlewareBinding.localAddress.getPort),
+      callbackUri = Uri("http://CALLBACK"),
```

Comment:
Shouldn’t this be `localhost/CALLBACK` or something like that?

Reply:
Good idea, I had taken the value from [here](https://github.com/digital-asset/daml/pull/8244/files/4e09651c237cd6d5d1b709187f3216bef7a9e8ab#diff-d3230e5e30d7ebb35124393b5718e52cf4460667e88e25177086088e5d2d1eeeL240) but using localhost is probably less confusing. I've made the login tests also consistently use `middlewareClient.loginUri`.